docker/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
+services:
+  snp_blending:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+      args:
+        - TAG=humble-master
+    environment:
+      DISPLAY: $DISPLAY
+      XAUTHORITY: $XAUTHORITY
+      # NVIDIA_DRIVER_CAPABILITIES: all
+      ROS_LOG_DIR: /tmp
+      ROS_LOCALHOST_ONLY: 1
+      ROS_DOMAIN_ID: 18
```

Comment:
```suggestion
      ROS_LOCALHOST_ONLY: $ROS_LOCALHOST_ONLY
      ROS_DOMAIN_ID: $ROS_DOMAIN_ID
```

Reply:
Made the change to lines 43 and 44 as well.